core/ginger.js

```diff
@@ -1,4 +1,5 @@
 const { CEC, CECMonitor } = require('@senzil/cec-monitor');
+const Seesaws = require('../obstacles/seesaw');
```

Comment:
Modules should best be loaded dynamically. Only on first time creation of seesaws should it be imported. +This is importing all seesaws. Probably best to split them up somehow.